src/System.Memory/tests/Span/Clear.cs

```diff
@@ -198,6 +199,31 @@ public static void ClearReferenceTypeLonger()
             Assert.Equal<string>(expected, actual);
         }
 
+        [Fact]
+        public static void ClearReferenceTypeSlice()
+        {
+            // A string array [ ""1", ..., "20" ]
+            string[] baseline = Enumerable.Range(1, 20).Select(i => i.ToString()).ToArray();
+
+            for (int i = 0; i < 16; i++)
+            {
+                // Going to clear array.Slice(1, i) manually,
+                // then compare it against array.Slice(1, i).Clear().
+                // Test is written this way to allow detecting overrunning bounds.
+
+                string[] expected = (string[])baseline.Clone();
+                for (int j = 0; j < i; j++)
+                {
+                    expected[j + 1] = null;
```

Comment:
Can we just start `j` with 1 within the for loop instead? Unless I am missing something.

Reply:
Fixed.